controllers/app.controllers.js

```diff
@@ -2,13 +2,25 @@ const models = require('../models/app.models');
 
 const getTopics = async (req, res, next) => {
 	try {
-		const results = await models.fetchTopics();
-		res.status(200).send({ topics: results });
+		const topics = await models.fetchTopics();
+		res.status(200).send({ topics: topics });
+	} catch (err) {
+		next(err);
```

Comment:
Some useful shorthand syntax for you:
- Because the key and value share the same name your response can be shortened down to `.send({ topics });`
- Because all you're doing in the catch block is invoking next you can shorten it down to `catch (next);`